plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1060,56 +1164,77 @@ void producer_plugin::plugin_initialize(const boost::program_options::variables_
+      EOS_ASSERT( _ro_thread_pool_size <= _ro_max_threads_allowed, plugin_config_exception,
+                  "read-only-threads (${th}) greater than the number of threads allowed (${allowed})",
+                  ("th", _ro_thread_pool_size) ("allowed", _ro_max_threads_allowed) );
 
-      my->_ro_write_window_time_us = fc::microseconds( options.at( "read-only-write-window-time-us" ).as<uint32_t>() );
-      my->_ro_read_window_time_us = fc::microseconds( options.at( "read-only-read-window-time-us" ).as<uint32_t>() );
-      EOS_ASSERT( my->_ro_read_window_time_us > my->_ro_read_window_minimum_time_us, plugin_config_exception, "read-only-read-window-time-us (${read}) must be at least greater than  ${min} us", ("read", my->_ro_read_window_time_us) ("min", my->_ro_read_window_minimum_time_us) );
-      my->_ro_read_window_effective_time_us = my->_ro_read_window_time_us - my->_ro_read_window_minimum_time_us;
+      _ro_write_window_time_us = fc::microseconds( options.at( "read-only-write-window-time-us" ).as<uint32_t>() );
+      _ro_read_window_time_us = fc::microseconds( options.at( "read-only-read-window-time-us" ).as<uint32_t>() );
+      EOS_ASSERT( _ro_read_window_time_us > _ro_read_window_minimum_time_us, plugin_config_exception,
+                  "read-only-read-window-time-us (${read}) must be at least greater than  ${min} us",
+                  ("read", _ro_read_window_time_us) ("min", _ro_read_window_minimum_time_us) );
+      _ro_read_window_effective_time_us = _ro_read_window_time_us - _ro_read_window_minimum_time_us;
 
       // Make sure a read-only transaction can finish within the read
       // window if scheduled at the very beginning of the window.
       // Add _ro_read_window_minimum_time_us for safety margin.
-      if ( my->_max_transaction_time_ms.load() > 0 ) {
-         EOS_ASSERT( my->_ro_read_window_time_us > ( fc::milliseconds(my->_max_transaction_time_ms.load()) + my->_ro_read_window_minimum_time_us ), plugin_config_exception, "read-only-read-window-time-us (${read} us) must be greater than max-transaction-time (${trx_time} us) plus ${min} us, required: ${read} us > (${trx_time} us + ${min} us).", ("read", my->_ro_read_window_time_us) ("trx_time", my->_max_transaction_time_ms.load() * 1000) ("min", my->_ro_read_window_minimum_time_us) );
+      if ( _max_transaction_time_ms.load() > 0 ) {
+         EOS_ASSERT( _ro_read_window_time_us > ( fc::milliseconds(_max_transaction_time_ms.load()) + _ro_read_window_minimum_time_us ),
+                     plugin_config_exception,
+                     "read-only-read-window-time-us (${read} us) must be greater than max-transaction-time (${trx_time} us) "
+                     "plus ${min} us, required: ${read} us > (${trx_time} us + ${min} us).",
+                     ("read", _ro_read_window_time_us) ("trx_time", _max_transaction_time_ms.load() * 1000)
```

Comment:
Some of these have a space between `)` and `(` while most do not.

Reply:
There were so many inconsistencies everywhere that I ended running clang-format and doing some manual cleanup.